yaml_test/pubspec.yaml

```diff
@@ -0,0 +1,23 @@
+name: yaml_test
+publish_to: none
+
+environment:
+  sdk: '>=2.2.0 <3.0.0'
+
+dev_dependencies:
+  build_runner: ^1.0.0
+  build_verify: ^1.1.0
+  test: ^1.6.0
+
+  # Repo packages
+  json_annotation: any
```

Comment:
If you intend these to always be path dependencies, I think you could just put that here and remove the overrides

Reply:
can't because it'll break transitive deps that assume `json_annotation` is a published (not path) dep.

Reply:
ack